recipes/fontconfig/all/conanfile.py

```diff
@@ -59,6 +67,8 @@ def _configure_autotools(self):
             args = ["--enable-static=%s" % ("no" if self.options.shared else "yes"),
                     "--enable-shared=%s" % ("yes" if self.options.shared else "no"),
                     "--disable-docs"]
+            if not self.options.with_nls:
+                args.append("--disable-nls")
```

Comment:
```suggestion
            args = [
                "--enable-static=%s" % ("no" if self.options.shared else "yes"),
                "--enable-shared=%s" % ("yes" if self.options.shared else "no"),
                "--enable-nls=%s" % ("yes" if self.options.shared else "no"),
                "--disable-docs"
            ]
```

Reply:
Thank you for the style suggestion. This is no longer required as I've removed the dependency on libgettext :)